examples/isentropic_example.py

```diff
@@ -0,0 +1,215 @@
+vwnd = data.variables['v_wind'][0, :] * units(data.variables['v_wind'].units)
+hgt = data.variables['Geopotential_height'][0, :] * units.meter
+spech = (data.variables['Specific_humidity'][0, :] *
+         units(data.variables['Specific_humidity'].units))
+
+#############################
+# To properly interpolate to isentropic coordinates, the function must know the desired output
+# isentropic levels. An array with these levels will be created below.
+
+isentlevs = [296.] * units.kelvin
+
+####################################
+# **Conversion to Isentropic Coordinates**
+#
+# Once three dimensional data in isobaric coordinates has been pulled and the desired
+# isentropic levels created, the conversion to isentropic coordinates can begin. Data will be
+# passed to the function as below. The function requires that isentropic levels, isobaric
+# levels, and temperature be input. Any additional inputs (in this case relative humidity, u,
+# and v wind components) will be linearly interpolated to isentropic space.
+
+isent_anal = mcalc.isentropic_interpolation(isentlevs,
+                                            lev,
+                                            tmp,
+                                            spech,
+                                            uwnd,
+                                            vwnd,
+                                            hgt,
+                                            tmpk_out=True)
+
+#####################################
+# The output is a list, so now we will separate the variables to different names before
+# plotting.
+
+
+isentprs = isent_anal[0]
+isenttmp = isent_anal[1]
+isentspech = isent_anal[2]
+isentu = isent_anal[3].to('kt')
+isentv = isent_anal[4].to('kt')
+isenthgt = isent_anal[5]
+
+########################################
+# A quick look at the shape of these variables will show that the data is now in isentropic
+# coordinates, with the number of vertical levels as specified above.
+
+print(isentprs.shape)
+print(isentspech.shape)
+print(isentu.shape)
+print(isentv.shape)
+print(isenttmp.shape)
+print(isenthgt.shape)
+
+#################################
+# **Converting to Relative Humidity**
+#
+# The NARR only gives specific humidity on isobaric vertical levels, so relative humidity will
+# have to be calculated after the interpolation to isentropic space.
+
+isentrh = mcalc.relative_humidity_from_specific_humidity(isentspech, isenttmp, isentprs)
+
+#######################################
+# **Plotting the Isentropic Analysis**
+
+
+# Set up our projection
+crs = ccrs.LambertConformal(central_longitude=-100.0, central_latitude=45.0)
+
+
+# Set up our array of latitude and longitude values and transform to
+# the desired projection.
+tlatlons = crs.transform_points(ccrs.PlateCarree(), lon, lat)
+tlons = tlatlons[:, :, 0]
+tlats = tlatlons[:, :, 1]
+
+# Limit the extent of the map area, must convert to proper coords.
+LL = crs.transform_point(-122., 22., ccrs.PlateCarree())
```

Comment:
In this case, with multiple plots using the same ax.set_extent, would it be better to only call crs.transform_point once (which is what this is doing, LL and UR are used in both plots)?